node-require/add.js

```diff
@@ -0,0 +1,3 @@
+const add = (x, y) => x + y;
+
+module.exports.add = add;
```

Comment:
I can see that you are doing `const add = require('./add')` in `calculate.js`, but I don't believe that the code will run. Is it possible that you made an additional commit that hasn't been pushed?

Reply:
No I don't think I did